### PR TITLE
added some comments to the code-review guide

### PR DESCRIPTION
**Pull request by nerik**
Added a few comments to the cod-review guide, to be discussed.

### CODE REVIEW

File: code-review/README.md [hunk 1]
```diff
@@ -26,12 +26,14 @@ Everyone
...
 * Be grateful for the reviewer's suggestions. ("Good call. I'll make that
   change.")
 * Don't take it personally. The review is of the code, not you.
 * Explain why the code exists. ("It's like that because of these reasons. Would
   it be more clear if I rename this class/file/method/variable?")
-* Extract some changes and refactorings into future tickets/stories.
+* Extract some changes and refactorings into future tickets/stories. As much as possible, a PR should either be a feature, or a technical PR, not both.
 * Link to the code review from the ticket/story. ("Ready for review:
   https://github.com/organization/project/pull/1")
 * Push commits based on earlier rounds of feedback as isolated commits to the
```
Comment (nerik) [inline code-review/README.md:39]: Squashing commits before merging is not actually something we seem to do at Vizz, but maybe we should discuss it (goes with the Changelog discussion we had with Agniezska)

Comment (simaob) [reply code-review/README.md:39]: Yes, that's true. I've tried to do it a bit sometimes, but I haven't done much programming to build a good habit of it. I think that commit squashing can be a nice thing to do and learning git is something good. =)

I'd propose having some kind of discussion/presentation on git practices and go from there into deciding.

Comment (simaob) [reply code-review/README.md:39]: Not sure what you mean by a "technical PR" in there.

Comment (agnessa) [reply code-review/README.md:39]: I understand it as stuff like "adds travis", "fixes formatting", "refactors all things"

Comment (simaob) [reply code-review/README.md:39]: OK. If it's clear, I'm happy to merge! =)

**Pull request merged**